### DISCUSSION TITLE
Why is my website not working?

### DISCUSSION THREAD

**Discussion by AbiThom** [community/community#70136] [New to GitHub] [answered]
As you can see in the screenshots of my repository, my website works fine on my computer, but when I deployed it to GitHub pages, it looks completely broken. How do I fix this? Sorry if this is a noob issue, I only started HTML coding about a week ago.

Here's what it looks like:
https://abithom.github.io/Website-Capstone/
How it looks to me:
https://github.com/AbiThom/Website-Capstone

**Comment by itsdotnet** [comment]
If you are using only HTML, CSS and JS
You can deploy it to Netlify
There is good ecosystem between Netlify and Github
You can easily deploy your codes with Github Repository

**Comment by radar-blip** [comment]
Try changing this link:
`<link rel="stylesheet" href="/css/global.css">`
to this: 
`<link rel="stylesheet" href="css/global.css">`

If that works, do the same for the other link (just remove the `/` in front of css).

**Comment by seyLu** [comment]
#### Hi @AbiThom,

Add a base href:
```diff
<head>
   ...
+  <base href="/Website-Capstone/">
</head>
``` 

And convert links to relative to the current page:
```diff
- <a ... href="/index.html">Home</a>
+ <a ... href="index.html">Home</a>
``` 

You can read more at:
- [MDN docs: base url elem](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)

**Comment by AbiThom** [comment]
i tried these methods but nothing worked. thanks for the help in advance!

**Reply by seyLu** [reply, marked as answer]
Make sure you have the `base url` set up and change every link to be relative to the current page.

```html
<!-- This will not work unless links are relative to the current page -->
<base href="/Website-Capstone/">
```

![image](https://github.com/community/community/assets/98249191/8b359e65-b172-4086-98cb-cf8eaae7ce91)
